workflow/src/test/java/com/synopsys/integration/alert/workflow/message/NotificationReceiverTest.java

```diff
@@ -93,11 +93,12 @@ private DistributionEvent createDistributionEvent(Long providerConfigId) throws
         LinkableItem subTopic = new LinkableItem("subTopic", "sub topic", null);
```

Comment:
Should this be renamed as well?

Reply:
I see it in a lot of the tests here.

Reply:
Yeah, there's another ticket to refactor usages of the words "topic" and "subTopic" elsewhere. This PR is primarily just getting the variables of this class refactored since refactoring touches so many files as it is.